assets/custom/css/contact.css

```diff
@@ -1,7 +1,78 @@
-/* mobile */
 
-/* desktop */
-@media screen and (min-width: 768px) {}
 
-/* tablet */
-@media screen and (min-device-width: 480px) {}
\ No newline at end of file
+/* tablet*/
+@media screen and (min-device-width: 480px) {}
+
+/* desktop */
+@media screen and (min-device-width: 1024px) {
+    .get_in_touch {
+        border-top: 1px solid rgba(51, 50, 61, 0.15);
+        border-bottom: 1px solid rgba(51, 50, 61, 0.15);
+        display: flex;
+        height: 50%;
+        padding-top: 2em;
+        padding-bottom: 2em;
+      
+    }
+    
+    .contact_me {
+        display: flex;
+        flex-direction: row;
+    }
+    
+    label {
+        font-family: Public Sans;
+        font-size: 13px;
+        font-weight: 700;
+        line-height: 30px;
+        letter-spacing: 0px;
+        text-align: left;
+    
+    }
+    p{
+        padding-left: 30%;
+    }
+    
+    input {
+        background-color: rgba(51, 50, 61, .1);
+        align-self: flex-end;
+        border: transparent;
+        Top: 9px;
+        Left: 16px;
+        Opacity: 40%;
+        height: 48px;
+        margin-bottom: 2em;
+        width: 635px;
+        padding-left: 2%;
+    }
+    
+    em {
+        width: 100%;
+        font-family: Ibarra Real Nova;
+        font-size: 32px;
+        font-weight: 700;
+        line-height: 42px;
+        letter-spacing: -0.3571428656578064px;
+        text-align: left;
+        color: rgba(51, 50, 61, 1);
+    
+    
+    }
+    
+    #message {
+        height: 96px;
+        width: 635px;
+        left: 0px;
+        top: 0px;
+        border-radius: 0px;
+    
+    
+    }
+    
+    .send {
+        background: rgba(32, 58, 76, 1);
+        color: rgba(255, 255, 255, 1);
+    
+    
+    }
+}
```

Comment:
Kindly leave a line after the closing curly bracket

Reply:
@devBoison there's still 17 commits in your PR